src/main/java/hudson/plugins/ansicolor/AnsiHtmlOutputStream.java

```diff
@@ -466,7 +469,7 @@ else switch (attribute) {
             closeTagOfType(AnsiAttrType.OVERLINE);
             break;
         default:
-            throw new IllegalStateException("Attribute " + attribute + " should not be reached");
+            break;
```

Comment:
See https://github.com/jenkinsci/ansicolor-plugin/issues/112#issuecomment-322742298 for the full analysis: https://github.com/jenkinsci/ansicolor-plugin/commit/2c09afa6729dc952a8fb99eeb9f23c15ae37cfdc changed the behavior to throw an IllegalStateException instead of ignoring unknown escape sequences. As only a fraction of existing escape sequences are handled, that is a very bad idea.